volmdlr/wires.py

```diff
@@ -3162,7 +3162,7 @@ def __eq__(self, other_):
         if self.__class__.__name__ != other_.__class__.__name__:
             return False
         equal = True
-        for edge, other_edge in zip(self.primitives, other_.edges):
+        for edge, other_edge in zip(self.primitives, other_.primitives):
```

Comment:
Edge and other_edge could be renamed primitive and other_primitive, feels more natural. 
I'd suggest to check inside the loop if equal is False, because in this case the loop can be broken, and False can be returned.

Reply:
Both changes are [here](https://github.com/Dessia-tech/volmdlr/blob/9265da2946a5625a32bfe870219ed4e710375e19/volmdlr/wires.py#L3165)